lib/rules/id-match.js

```diff
@@ -249,6 +267,10 @@ module.exports = {
 
                 const isClassField = node.parent.type === "PropertyDefinition";
 
+                if (isReferenceToGlobalVariable(node)) {
+                    return;
+                }
```

Comment:
This seems unnecessary, private identifier can never be a reference to a variable?

Reply:
Yes, this is redundant. My bad :)